Dockerfile

```diff
@@ -10,21 +10,33 @@ ARG WAUTH_VERSION_ARG
 COPY go.mod .
 COPY go.sum .
 
+# Running upgrades
+RUN apk update && apk upgrade
+
+# Install NPM for MJML
+RUN apk add --update npm
+
 # Download dependencies
 RUN go mod download
 
 # Copy source
 COPY . .
 
+# Initialising NPM environment for MJML
+RUN npm init -y && npm install mjml
+
+# Generate the mjml files to tmpl
+RUN go generate
```

Comment:
May just be me, but I'd prefer to generate them and commit them to git, and then just regenerate them if it changes - one fewer moving part in the build process (see also https://go.dev/blog/generate).

Reply:
I have done this change